bn_s_mp_exptmod_fast.c

```diff
@@ -85,7 +85,7 @@ int s_mp_exptmod_fast(const mp_int *G, const mp_int *X, const mp_int *P, mp_int
       /* automatically pick the comba one if available (saves quite a few calls/ifs) */
 #ifdef BN_S_MP_MONTGOMERY_REDUCE_FAST_C
       if ((((P->used * 2) + 1) < (int)MP_WARRAY) &&
-          (P->used < (1 << ((CHAR_BIT * sizeof(mp_word)) - (2 * MP_DIGIT_BIT))))) {
+          (P->used < (1 << MP_MAXFAST_BIT))) {
```

Comment:
Would it make sense to introduce another macro MP_MAXFAST for (1 << MP_MAXFAST_BIT)?